Stuck in state "Pending"

Hello,

I have just created a docker swarm from scratch, and I am not able to run the visualizer, since its stuck in pending.

```
$ docker service ls && docker service ps viz
ID                  NAME                MODE                REPLICAS            IMAGE                             PORTS
ylpdkfqjv0o8        viz                 replicated          0/1                 dockersamples/visualizer:latest   *:9090->8080/tcp
ID                  NAME                IMAGE                             NODE                DESIRED STATE       CURRENT STATE           ERROR               PORTS
j7z2l72rlf46        viz.1               dockersamples/visualizer:latest                       Running             Pending 4 minutes ago                       
```
Docker version:
```
$ docker version
Client:
 Version:      17.05.0-ce
 API version:  1.29
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 22:10:54 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.05.0-ce
 API version:  1.29 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 22:10:54 2017
 OS/Arch:      linux/amd64
 Experimental: false

```
``` 
apt-cache policy docker-engine
docker-engine:
  Installed: 17.05.0~ce-0~ubuntu-xenial
  Candidate: 17.05.0~ce-0~ubuntu-xenial
``` 

Same setup on 17.03 works, so I suspect the 17.05 is not supported yet?

OK inspect was showing that node couldn't find `healthcheck.js` for some reason:

```
{
                        "Start": "2017-12-31T21:23:14.536991044Z",
                        "End": "2017-12-31T21:23:14.849570942Z",
                        "ExitCode": 1,
                        "Output": "module.js:487\n    throw err;\n    ^\n\nError: Cannot find module '/healthcheck.js'\n    at Function.Module._resolveFilename (module.js:485:15)\n    at Function.Module._load (module.js:437:25)\n    at Function.Module.runMain (module.js:605:10)\n    at startup (bootstrap_node.js:158:16)\n    at bootstrap_node.js:575:3\n"
                    }
```

Putting in a relative path didn't help, but hardcoded full path worked. PR coming.

@BretFisher Got the same issue: `docker inspect` showing _Cannot find module '/healthcheck.js'_ on a service. I use 
 dockersamples/visualizer@sha256:868b83d5fb2aff628f6dfaeb081b0e0124cdaf06354bf252d92d878b75193f7e

--- a/Dockerfile
+++ b/Dockerfile
@@ -34,6 +34,6 @@ ENV MS=1000 CTX_ROOT=/
 
 EXPOSE 8080
 
-HEALTHCHECK CMD node healthcheck.js || exit 1
+HEALTHCHECK CMD node /app/healthcheck.js || exit 1
 
 CMD ["npm","start"]

